docs/cf-cli-plugin.md

```diff
@@ -0,0 +1,210 @@
+# Plug-in for the “Cloud Foundry CLI”
+“Application Autoscaler” offers a plug-in for the “Cloud Foundry CLI”. With it users can attach or detach scaling-policies to their apps. They can view them and view the scaling-history. And they can view the metrics send by their apps.
+
+This is useful to get some insights on how currently applied scaling-policies work and if changes are reasonable. It is as well useful for debugging. A common case is, when [custom-metrics](<../defining-a-custom-metric-87e657e.md>) are introduced to scale an app and the developers want to check, if the send metrics are correctly processed by Autoscaler and why up-/down-scaling happens or does not happen.
+
+## Command List
+| Command                                                          | Description                                    |
+|------------------------------------------------------------------|------------------------------------------------|
+| [autoscaling-api, asa](#cf-autoscaling-api)                      | Set or view AutoScaler service API endpoint    |
+| [autoscaling-policy, asp](#cf-autoscaling-policy)                | Retrieve the scaling policy of an application  |
+| [attach-autoscaling-policy, aasp](#cf-attach-autoscaling-policy) | Attach a scaling policy to an application      |
+| [detach-autoscaling-policy, dasp](#cf-detach-autoscaling-policy) | Detach the scaling policy from an application  |
+| [autoscaling-metrics, asm](#cf-autoscaling-metrics)              | Retrieve the metrics of an application         |
+| [autoscaling-history, ash](#cf-autoscaling-history)              | Retrieve the scaling history of an application |
+
+## Command usage
+### `cf autoscaling-api`
+Set or view the API-endpoint of the service Autoscaler to be used by this CLI. If the CF API endpoint is <https://api.example.com>, then typically the Autoscaler-API-endpoint will be https://autoscaler.example.com. Check the manifest when Autoscaler is deployed to get the API-endpoint.
+
+> ### Note:
+> This command is usually not needed when working with BTP-Cloud-Foundry.
```

Comment:
See https://brand.sap.com/naming-center/search?keywords=cloud%2520foundry&status=APPROVED
```suggestion
> This command is usually not needed when working with SAP BTP, Cloud Foundry environment.
```

Reply:
Didn't know this as well.